_posts/2016-09-10-stripe-dress.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: The psychology of perception tests an age-old fashion foible
+author: neuroamanda
+image:
+  feature: 2016-09-10-stripe-dress.jpg
+  credit: Gareth Williams
+  caption: Horizontal-striped dress
+---
+
+_Horizontal stripes on clothing are in fact slimming_
+
+[The dress](http://wp.me/p2UE9j-1da) sparked one of the hottest fashion debate of the 21st century, but another mystery has occupied scientists and fashionistas alike since the mid-19th century: do horizontal stripes make you look fat?
+The answer may seem obvious, but experimental results and a well-documented illusion contradict many people's intuition that horizontal stripes make the wearer look wider.
+
+<!--break-->
+
+As pioneering vision scientist Helmholtz [noted back in 1867](http://poseidon.sunyopt.edu/helmholtz/OCRVolume3.pdf) (see page 206!), a person wearing horizontal stripes should, contrary to popular belief, appear _thinner_ than someone with vertical stripes.
+This observation is the basis for the [Helmholtz illusion](http://www.bbc.co.uk/radio4/features/sywtbas/finalists/stripes/), at least as it applies to shapes, not bodies.
+A team of researchers from Japan tested whether the size of the body wearing stripes changes the strength of the effect.
+
+[In line with previous research](http://ipe.sagepub.com/content/2/1/69), they found that the common idea that horizontal stripes make one look fat is in fact wrong.
```

Comment:
Personally, I don't like having "in fact" outside of parenthetical commas, but that's just me (see also the dek).